v-next/hardhat/test/internal/hre-intialization.ts

```diff
@@ -26,6 +27,52 @@ describe("HRE intialization", () => {
   });
 
   describe("createHardhatRuntimeEnvironment", () => {
+    describe("project root", () => {
+      describe("no path param is passed", () => {
+        it("should use the CWD as the default path", async () => {
```

Comment:
This isn't the expected behavior. What happens is that it looks for the closest package.json starting from the CWD, and uses its directory.

Reply:
The tests have been modified